### PR TITLE
Return of the swords PR, now with 25% more swords

### PR DESCRIPTION
**Pull request by Mortiz999**
## About The Pull Request

Adds the Langesmesser and Estoc as new two steel cost alternatives to the bastard sword. Allows for the 1h messer to be made with steel. Also fixes the steel short sword, reworking it into an arming sword for weaker chars/squires.

## Why It's Good For The Game
The goal of this pr is to create three 'classes' for swords, of both short and medium length, creating more weapon diversity. Cutting swords, such as the langesmesser and sabre, Stabbing swords, like the rapier and estoc, and a set of all rounders, the arming sword and bastard sword.

For the steel messer, if it can be made out of iron, it can be made out of steel, and the new steel messer is still weaker than the sabre. It's only saving grace is it's durability and style. The arming sword acts as a middle ground between the rapier and sabre, with it's ability to bash setting it apart. Also included is a description for the eaglebeak, to fix its lack of one.

### CODE REVIEW

**Comment by Teragenic** [comment]
Please also include pictures and sprites of the swords. Including if there are any runtimes while compiling or if there are none.

**Comment by Mortiz999** [comment]
OOPS
Here are the swords in question, making up the bottom row. Already in-game swords are on the top row for reference. In order: Estoc, Langesmesser, Steel Messer, and Arming Sword. I am planning on reworking the Langesmesser, as I don't love the sprite I made for it.
I haven't had runtimes with loading or smithing the swords, so I assume it runs properly.

**Comment by Teragenic** [comment]
There's still the issue of the armor buff not being mentioned anywhere

File: code/modules/clothing/rogueclothes/armor.dm [hunk 1]
```diff
@@ -191,7 +191,7 @@
 	icon_state = "brigandine"
 	blocksound = SOFTHIT
 	body_parts_covered = CHEST|GROIN|VITALS|LEGS|ARMS
-	armor = list("blunt" = 90, "slash" = 60, "stab" = 70, "bullet" = 100, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	armor = list("blunt" = 90, "slash" = 90, "stab" = 70, "bullet" = 100, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```
[the remark this answers is not in the document]
Comment (Mortiz999) [reply code/modules/clothing/rogueclothes/armor.dm:194]: Will be reverting that, forgot that I left it in after messing around with DMI

[the remark this answers is not in the document]
Comment (Mortiz999) [reply code/modules/clothing/rogueclothes/armor.dm]: Reverted now, thanks for catching it!

**Comment by coffeecoffeecoffee** [comment]
If I may ask, what are their statistical differences and what differentiates ALL 8 of the different swords?

**Comment by Mortiz999** [comment]
> If I may ask, what are their statistical differences and what differentiates ALL 8 of the different swords?

Only 4 were added, the top row are in-game as of now. Stats wise:
The arming sword is identical in damage to the sabre, but with different intents, meaning a slower swing speed
The steel messer is the same as the iron one, with more durability, mainly added for smith's convenience
The Langesmesser is a cutting/two handed focused sword, similar to the desert rider's sword. It's saving grace, compared to the bastard sword, is the better CHOP intent it borrows from the shorter messer, which does good damage to armor, and can serve as a good finishing move.
The Estoc is a specialized sword, which shares some of its' traits with spears, mainly their bad cutting stats, and ability to do damage when thrusting at armor. Compared to the bastard sword, it struggles being used one-handed, and is arguably more useful when fighting armored opponents.

**Comment by Mortiz999** [comment]
The new sprites for the langesmesser didn't embed for the change, so here they are:
Screenshot_56
Screenshot_55
Not sure if they need tweaking, glad to change it if they do.

**Pull request closed without merging**